instrumentation/sql/vendor_mysql.go

```diff
@@ -25,7 +25,7 @@ func (ext *mysqlExtension) ProcessConnectionString(connectionString string, conf
 	configuration.port = dsn["Port"]
 	configuration.instance = dsn["DBName"]
 	configuration.host = dsn["Host"]
-	configuration.connString = strings.ReplaceAll(connectionString, dsn["Passwd"], "******")
+	configuration.connString = strings.Replace(connectionString, dsn["Passwd"], "******", -1) // ReplaceAll is not supported on go1.11
```

Comment:
I don't think we need the comments here

Reply:
I'll remove all the `ReplaceAll` comments in the PR